### DISCUSSION TITLE
Documentation issues

### DISCUSSION THREAD

**Discussion by lishine** [LegendApp/legend-state#263] [General]
Can we add this nuance to doc?
https://github.com/LegendApp/legend-state/issues/118#issuecomment-1562257965

I can do a PR

**Comment by jmeistrich** [comment]
Yes a PR would be great! Docs are in this repo: https://github.com/LegendApp/legend-docs

**Comment by lishine** [comment]
more doc

This is confusing:
`return <div>Sum: {sum}</div>;`
Because sum is an observable, it does do anything in a div ...

<img width="905" alt="image" src="https://github.com/LegendApp/legend-state/assets/6741645/2260f5bc-12e7-46fe-85aa-8f2b80072818">

**Comment by lishine** [comment]
I have no words to describe how wonderfull all the tools of this lib!
Just what I would really need! Such as these:
`useVerifyOneRender()`
`useVerifyNotTracking`

**Comment by lishine** [comment]
This page says:
>Call useTraceListeners() anywhere within `observe` or any React component
>Call useTraceUpdates() anywhere within `observe` or any React component

But it should be `observer`

**Comment by lishine** [comment]
Dark/light mode resets on page transition. Also not connected to system settings. 

Header on mobile messed up.

**Comment by lishine** [comment]
Link broken in here
>See Tracking for more about when it tracks

**Comment by lishine** [comment]
https://github.com/LegendApp/legend-state/issues/296#issuecomment-2158034555
